javascript/javascript1/week2/student_manager.js

```diff
@@ -0,0 +1,36 @@
+const class07Students = new Array();
+
+
+
+//var totalStudents = addStudentToClass("ARNAB");
+//console.log("This variable holds -> " + totalStudents);
+//addStudentToClass("PALLABI");
+
+function addStudentToClass(actualStudentName) {
+    if (actualStudentName == "QueenofDenmark") {
+        class07Students.push(actualStudentName)
+        console.log("1st pos:" + class07Students)
+        return ("you can always add QD")
```

Comment:
We don't want to return a string. Push the name into the array `class07Students`

When you return. The function will stop here and don't continue. So the next lines of code will be unreachable